### ISSUE TITLE
How can I capture the event when a field is formatted?

### ISSUE THREAD

**Issue by tkiethanom** [itsjavi/bootstrap-colorpicker#145]
For example if I type `asdf` into the field it gets formatted to `#000000`. The I have an onChange event on the field but it returns `asdf` but I want the `#000000`.

**Comment by tkiethanom** [comment]
Adding a delay with setTimeout seems to work but there should be a cleaner way.

**Comment by mjolnic** [comment]
there is no "onFormat" event, what are you trying to accomplish?

**Comment by tkiethanom** [comment]
I needed to fire a function when the field was changed, but like I said earlier if the field gets formatted from `asdf` to `#000000` the onChange event would receive the value prior to being reformatted (`asdf`). I realized the formatting is happening once the focus leaves the field so now I'm using the onBlur event instead of onChange. 

Ignore that part about setTimeout, that was a bad way of doing it.
